<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*主空间区域*/
#Main {
    background-color: white;
    border-radius: 8px;
}

/*关闭按钮*/
#BtnClose {
    border: none;
    padding: 4px;
    border-radius: 4px;
    background-color: white;
}

#BtnClose:hover {
    border: none;
    padding: 4px;
    border-radius: 4px;
    background-color: red;
}

/*最小化按钮*/
#BtnMinimize {
    border: none;
    padding: 4px;
    border-radius: 4px;
    background-color: white;
}

#BtnMinimize:hover {
    border: none;
    padding: 4px;
    border-radius: 4px;
    background-color: aliceblue;
}

/*表格*/
#ImageTable {
    border: 1px solid #8CCCF4;
    border-radius: 8px;
    padding: 8px;
    background-color: #8CCCF4;
    outline: none;
}

#ImageTable::item {
    color: #8CCCF4;
    background-color: aliceblue;
    border: 1px solid #8CCCF4;
    border-radius: 8px;
    margin: 4px;
    outline: none;
}

#ImageTable::item:selected {
    color: #51B6F4;
}

QHeaderView {
    background: transparent;
    font-weight: bold;
}

QHeaderView::section {
    color: #8CCCF4;
    background-color: aliceblue;
    border-radius: 8px;
    margin: 4px;
    outline: none;
}

QHeaderView::section:checked {
    color: #51B6F4;
}

/*功能按钮*/
#BtnDeleteImage,
#BtnOpenImage,
#BtnSaveImage,
#BtnUpdateDB,
#BtnUploadDB {
    border: none;
    border-radius: 8px;
    min-width: 80px;
    margin-left: 8px;
    padding: 8px;
    color: aliceblue;
    background-color: #8CCCF4;
}

#BtnDeleteImage:hover,
#BtnOpenImage:hover,
#BtnSaveImage:hover,
#BtnUpdateDB:hover,
#BtnUploadDB:hover {
    border: none;
    border-radius: 8px;
    min-width: 80px;
    margin-left: 8px;
    padding: 8px;
    color: aliceblue;
    background-color: #74CCF4;
}

#BtnDeleteImage:pressed,
#BtnOpenImage:pressed,
#BtnSaveImage:pressed,
#BtnUpdateDB:pressed,
#BtnUploadDB:pressed {
    border: none;
    border-radius: 8px;
    min-width: 80px;
    margin-left: 8px;
    padding: 8px;
    color: aliceblue;
    background-color: #51B6F4;
}

#BtnDeleteImage:focus,
#BtnOpenImage:focus,
#BtnSaveImage:focus,
#BtnUpdateDB:focus,
#BtnUploadDB:focus {
    outline: none;
}

/*图片Label*/
#Image {
    color: #8CCCF4;
    background-color: transparent;
}

QScrollBar:horizontal {
    height: 12px;
    padding: 2px;
    background: aliceblue;
    margin: 0px, 0px, 0px, 0px;
    border-radius: 6px;
}

QScrollBar::handle:horizontal {
    height: 8px;
    background: #8CCCF4;
    border-radius: 4px;
}

QScrollBar::handle:horizontal:hover {
    height: 8px;
    background: #51B6F4;
    border-radius: 4px;
}

QScrollBar::add-line:horizontal,
QScrollBar::sub-line:horizontal {
    height: 0px;
    width: 0px;
}

QScrollBar::add-page:horizontal,
QScrollBar::sub-page:horizontal {
    background: aliceblue;
}

QScrollBar:vertical {
    width: 12px;
    padding: 2px;
    background: aliceblue;
    margin: 0px, 0px, 0px, 0px;
    border-radius: 6px;
}

QScrollBar::handle:vertical {
    width: 8px;
    background: #8CCCF4;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
    width: 8px;
    background: #51B6F4;
    border-radius: 4px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
    width: 0px;
}

QScrollBar::add-page:vertical,
QScrollBar::sub-page:vertical {
    background: aliceblue;
}

#BtnSearch {
    border: none;
    padding: 4px;
    color: #8CCCF4;
    background-color: transparent;
}

#Input {
    border: 1px solid rgba(140, 204, 244, 128);
    border-radius: 8px;
    min-width: 80px;
    margin-left: 8px;
    padding: 8px;
    color: #8CCCF4;
    selection-color: white;
    background-color: white;
    selection-background-color: #8CCCF4;
}

#Input:focus {
    border: 1px solid #8CCCF4;
    border-radius: 8px;
    min-width: 80px;
    margin-left: 8px;
    padding: 8px;
    color: #8CCCF4;
    background-color: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QFrame" name="Main">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" name="Caption" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="BtnOpenImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>打开图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnSaveImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>保存图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnDeleteImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
             <stylestrategy>PreferDefault</stylestrategy>
            </font>
           </property>
           <property name="text">
            <string>删除图片</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnUploadDB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>上传数据库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnUpdateDB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>更新数据库</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="Input">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="QToolButton" name="BtnSearch">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="res/res.qrc">
             <normaloff>:/img/img/search.png</normaloff>:/img/img/search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="BtnMinimize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>最小化</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res/res.qrc">
             <normaloff>:/img/img/min.png</normaloff>:/img/img/min.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnClose">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="whatsThis">
            <string>关闭
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="res/res.qrc">
             <normaloff>:/img/img/close.png</normaloff>:/img/img/close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="rightMargin">
         <number>8</number>
        </property>
        <item>
         <widget class="QTableWidget" name="ImageTable">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="ImageBottom">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="topMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QLabel" name="Image">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>还没有图片哦~</string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnClose</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>728</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnMinimize</sender>
   <signal>clicked()</signal>
   <receiver>Window</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
